### PR TITLE
Updated html-forms.md

### PR DESCRIPTION
**Pull request by imanishbarnwal**
Added a few resource links in Independent Practice.
Fixed issue: #1364

### CODE REVIEW

**Comment by imanishbarnwal** [comment]
> @vegetabill are the echo/debug links relevant for what you had in mind?

Have you checked my PR?

File: web/html-forms.md [hunk 1]
```diff
@@ -214,7 +214,11 @@ Only basic knowledge of HTML is required to create a form in HTML. You would nee
 ### Independent Practice
 Spend 15 minutes researching these topics online:
 - Learn about what happens when an HTML form is submitted. 
+    - [Sending Form Data By Mozilla](https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data)
+    - [HTML Forms By w3schools](https://www.w3schools.com/html/html_forms.asp)
 - Are there echo/debug servers that can print out what is submitted to the server?
+    - [PHP Form Handling](https://www.w3schools.com/php/php_forms.asp)
+    - [PHP Debugging Tips](https://stackify.com/php-debugging-tips/)
```
Comment (vegetabill) [inline web/html-forms.md:221]: Please remove these links as they are not relevant (we do not cover PHP in this curriculum)

**Comment by vegetabill** [comment]
> @vegetabill are the echo/debug links relevant for what you had in mind?

Nope but I spent 10min googling and couldn't find anything so let's just drop them.

**Comment by alodahl** [comment]
> > @vegetabill are the echo/debug links relevant for what you had in mind?
> 
> Nope but I spent 10min googling and couldn't find anything so let's just drop them.

Are you saying we should drop the echo/debug question too?

**Comment by alodahl** [comment]
@imanishbarnwal once you remove the two links that @vegetabill mentioned, you can pick up a new issue while we wait for his response.

**Comment by imanishbarnwal** [comment]
> @imanishbarnwal once you remove the two links that @vegetabill mentioned, you can pick up a new issue while we wait for his response.

Yeah sure @alodahl, kindly check them now. I just removed those links.

**Comment by Kundan28** [comment]
Updated the scoresheet.

**Pull request merged**